BUILD

```diff
@@ -46,6 +46,7 @@ genrule(
     name = "android_dist",
     srcs = ["//:android_aar"],
     outs = ["android_out"],
+    stamp = True,
```

Comment:
do we want to stamp `ios_dist` as well?

Reply:
We're good because that's happening on iOS already for other reasons https://github.com/envoyproxy/envoy/blob/5dd7cc99af303cb92741718836ad4e73639d630f/source/common/common/BUILD#L262-L265

Reply:
Ah actually I don't think that other one is enough, so yes I added it for that too